### DISCUSSION TITLE
Corrupted beatmaps

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#23855] [Q&A]
Lots of my beatmaps are corrupted is this a known issue
95% are fine but the rest is "unreadable"

https://github.com/ppy/osu/assets/40531871/3c421483-8de6-4afa-8fd7-42ca0d83d9a8

[database.log](https://github.com/ppy/osu/files/11708754/database.log)
[input.log](https://github.com/ppy/osu/files/11708755/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11708756/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/11708757/network.log)
[performance.log](https://github.com/ppy/osu/files/11708758/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/11708759/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/11708760/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/11708761/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/11708762/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/11708763/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11708764/updater.log)

**Comment by bdach** [comment]
How did you import these?

**Reply by AuracleTech** [reply]
These were maps since 2021~ I just move the whole osu folder around, not sure why they got corrupted
